package.json

```diff
@@ -47,12 +47,15 @@
         "stylelint-config-standard": "^36.0.1",
         "stylelint-config-tailwindcss": "^0.0.7",
         "svelte": "^4.2.19",
-        "svelte-check": "^4.0.1",
-        "svelte-eslint-parser": "^0.41.0",
+        "svelte-check": "^4.0.2",
+        "svelte-eslint-parser": "^0.41.1",
         "tailwindcss": "^3.4.11",
         "terser": "^5.32.0",
         "typescript": "^5.6.2",
         "typescript-eslint": "^8.5.0",
-        "vite": "^5.4.4"
+        "vite": "^5.4.5"
+    },
+    "dependencies": {
+        "smooth-scroll": "^16.1.3"
```

Comment:
This dependency is no longer needed with the native CSS solution that I proposed.